publications.html

```diff
@@ -0,0 +1,200 @@
+    <link rel="stylesheet" href="stylesheets/github-light.css">
+    <meta name="viewport" content="width=device-width">
+</head>
+
+<body class="site">
+<header class="site-header">
+    <nav>
+        <div class="sticky-nav">
+            <div class="container-nav">
+                <div class="nav-container">
+                    <div class="navbar-brand"><a href="http://cldf.clld.org">
+                        <img width="150" src="logos/logo_straight.png" alt="CLDF">
+                    </a></div>
+                    <div class="navbar-nav nav-mobile">
+                        <a class="nav-item nav-link btn btn-nav" href="/">Home</a>
+                        <a class="nav-item nav-link btn btn-nav"
+                           href="https://github.com/cldf/cldf">Specification</a>
+                        <a class="nav-item nav-link btn btn-nav" href="v1.0/terms.rdf">Ontology</a>
+                        <a class="nav-item nav-link btn btn-nav" href="/publications.html">Publications</a>
+                        <a class="nav-item nav-link btn btn-nav" href="/examples.html">Examples</a>
+                    </div>
+                    <div class="navbar-nav nav-main">
+                        <a class="nav-item nav-link" href="/">Home</a>
+                        <a href="https://github.com/cldf/cldf" class="nav-item nav-link">Specification</a>
+                        <a href="v1.0/terms.rdf" class="nav-item nav-link">Ontology</a>
+                        <a class="nav-item nav-link active"
+                           href="/publications.html">Publications</a>
+                        <a class="nav-item nav-link" href="/examples.html">Examples</a>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </nav>
+</header>
+<main class="site-content">
+    <section class="pad-top">
+        <div class="container">
+            <h2 class="headline text-center">CLDF-related Publications</h2>
+        </div>
+    </section>
+    <section class="section-main">
+        <div class="container pad-top">
+            <div class="service-flexrow">
+                <div class="column-50 gray-box">
+                    <h2>General papers</h2>
+                    <ul>
+                        <li><a href="https://digling.org/evobib/?bibtex=Forkel2018a">Cross-Linguistic
+                            Data Formats, advancing data sharing and re-use in comparative
+                            linguistics</a></li>
+                    </ul>
+                </div>
+                <div class="column-50 gray-box">
+                    <h2>Catalogs</h2>
+                    <h3>Glottolog</h3>
+                    <ul>
+                        <li>Paper</li>
+                        <li>Data:
+                            <a href="https://doi.org/10.5281/zenodo.3554959"><img
+                                    src="https://zenodo.org/badge/DOI/10.5281/zenodo.3554959.svg"
+                                    alt="DOI"></a>
```

Comment:
I'd add short descriptions of what to expect/find, e.g. "Glottolog data releases on Zenodo", or "pyglottolog Python library"